pypit/tests/test_objfind.py

```diff
@@ -0,0 +1,65 @@
+# Module to run tests on ararclines
+
+
+import os
+import numpy as np
+import pytest
+
+from astropy.table import Table
+
+from pypit import pyputils
+msgs = pyputils.get_dummy_logger()
+from pypit import arutils
+from pypit import artrace
+from pypit import arcytrace
+
+
+def data_path(filename):
+    data_dir = os.path.join(os.path.dirname(__file__), 'files')
+    return os.path.join(data_dir, filename)
+
+
+def test_very_close_obj():
+    """ Find objects with minima algorithm
+    """
+    # Read trace example
+    tbl = Table.read(data_path('real_close_trace.fits'))
+    trcprof = tbl['trc'].data.astype(np.float64)
+    # Call
+    objl, objr, bckl, bckr = artrace.find_obj_minima(trcprof, fwhm=3., nsmooth=0)
+    assert objl[1] == 55
```

Comment:
Should we assert the other parameters returned from this algorithm?

(likewise with `test_objstd` and `test_objmin`)